andela/technites-bn-frontend
# #168049938 Run codeclimate test coverage

#### What does this PR do?
Run code climate test coverage
#### Description of Task to be completed?
- Adding `technites-bn-frontend` repo to Code Climate
- Adding `technites-bn-frontend` repo to CircleCI
- Add `.cicleci/config.yml` file
- Add Code Climate Test Reporter ID to `.circleci/config.yml file`
#### How should this be manually tested?
- Check `.circleci/config.yml` file in the source code and check the build on this link
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#168049938](https://www.pivotaltracker.com/story/show/168049938)
#### What are the relevant Github Issues?
N/A
#### Screenshots (if appropriate)
N/A
#### Questions: N/A
